docs/_posts/2020-05-28-fastest-bert-training.md

```diff
@@ -92,6 +92,32 @@ throughput by combining our software optimizations with the new hardware. We
 project it would reduce BERT training time further to less than 25 minutes on a
 cluster of 1024 A100 GPUs.
 
+## Performance Results for Fine-Tuning Tasks
+
+In addition to the performance benefits we show for the pretraining, 
+we have evaluated the performance of our customized kernel for fine-tuning the 
+downstream tasks. Tables 2 and 3 show the samples-per-second achieved when running 
+Bing-BERT Squad on NVIDIA V100 using 16 and 32 GB of memory, using Pytorch and DeepSpeed transformer kernels. 
+For the 16-GB V100, we can achieve up to 1.5x speedup while supporting 2x larger batch size per GPU. 
+On the other hand, we can support as large as 32 batch size (2.6x more than Pytorch) using 32GB of memory, while providing 1.3x speedup for the end-to-end fine-tune training. Note, that we use the best 
+samples-per-second to compute speedup for the cases that PyTorch runs OOM.
+
+| Micro Batch size | PyTorch | DeepSpeed | Speedup (x) |
+| ---------------- | ------- | --------- | ----------- |
+| 4                | 36.34   | 50.76     | 1.4         |
+| 6                | OOM     | 54.28     | 1.5         |
+| 8                | OOM     | 54.16     | 1.5         |
+Table 2. Samples/second for running Squad Fine-Tuning on NVIDIA V100 (16-GB) using PyTorch and DeepSpeed transformer kernels.
```

Comment:
The table captions are showing up inside the table. Can we put the table captions outside?

Reply:
Sure, I will do that. Thanks